packages/google_maps_flutter/google_maps_flutter_web/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 0.4.1
+
+* Fires a `MapStyleException` when an invalid JSON is used in `setMapStyle` (was
+  `FormatException` previously).
```

Comment:
Shouldn't we mark this as a breaking change and version accordingly?

Reply:
We could, however this `_web` package isn't used directly by end users, right? I actually needed to do this change for a test in the core package to pass :)

Reply:
(Done)